src/main/java/thePackmaster/ui/FixedModLabeledToggleButton/FixedModLabel.java

```diff
@@ -0,0 +1,90 @@
+package thePackmaster.ui.FixedModLabeledToggleButton;
+
+import basemod.IUIElement;
+import basemod.ModPanel;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.g2d.BitmapFont;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.helpers.FontHelper;
+
+import java.util.function.Consumer;
+
+// This class is a near-copy of the BaseMod class, changed to use Settings.xScale and Settings.yScale
```

Comment:
In theory, it would be best if the versions of these classes in BaseMod were replaced with what I made here. (The code for those classes can be found at https://github.com/daviscook477/BaseMod/tree/master/mod/src/main/java/basemod).

In practice, mods using the BaseMod classes that rely on or work around the current functionality (i.e. using only `Settings.scale`, not `Settings.xScale` and `Settings.yScale`) could easily be broken by replacing these, and dealing with that beyond what I think is wise to take on when the immediate goal is just to fix this issue for Packmaster.